src/renderer/helpers/sponsorblock.js

```diff
@@ -56,3 +56,31 @@ export async function deArrowData(videoId) {
     throw error
   }
 }
+
+export async function deArrowThumbnail(videoId, timestamp) {
+  let requestUrl = `${store.getters.getDeArrowThumbnailGeneratorUrl}/api/v1/getThumbnail?videoID=` + videoId
+  if (timestamp != null) {
+    requestUrl += `&time=${timestamp}`
+  }
+
+  try {
+    const response = await fetch(requestUrl)
+
+    // 404 means that there are no segments registered for the video
```

Comment:
This comment should probably be updated to mention thumbnails instead of segments.